WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabViewModel.cs

```diff
@@ -135,6 +136,11 @@ public ReceiveTabViewModel(WalletViewModel walletViewModel)
 				{ }
 			}, isCoinListItemSelected);
 
+			ReLabelCommand = ReactiveCommand.Create(() =>
+			{
+				SelectedAddress.InEditMode = true;
+			}, isCoinListItemSelected);
```

Comment:
This modifies a UI element, so `, outputScheduler: RxApp.MainThreadScheduler` is needed.

Reply:
its fine to put it in there, but since a Commands are bound to UI Controls they are always raised on UIThread.

I dont think we should be putting this everywhere, because anywhere a command is not being raised on UIThread thats a bug, and this is just a workaround.

Reply:
I misunderstood what's this. I thought this makes sure the command runs on the UI thread, but that's not the case.

- This makes sure some other stuff runs on the UI thread after the command, not sure exactly what.
- As you said, the command is bound to the UI thread, so this wouldn't affect anything.
- It defaults to RxApp.MainThreadScheduler anyway.

So removing it.